### PR TITLE
Add optional param contentType to Blob.slice method

### PR DESCRIPTION
**Pull request by vobruba-martin**
This change is [Reviewable](https://reviewable.io/reviews/google/closure-compiler/1838)

### CODE REVIEW

File: externs/browser/fileapi.js [hunk 1]
```diff
@@ -50,30 +50,33 @@ Blob.prototype.type;
  * @see http://www.w3.org/TR/FileAPI/#dfn-slice
  * @param {number} start
  * @param {number} length
+ * @param {string=} contentType
```
Comment (MatrixFrog) [inline externs/browser/fileapi.js]: Since it's optional it should be called `opt_contentType`

**Comment by MatrixFrog** [comment]
Can you squash this into a single commit? That makes it a lot easier for us to import the change. I just added this as an instruction on https://github.com/google/closure-compiler/wiki/Contributors

**Comment by MatrixFrog** [comment]
lgtm other than that

**Comment by vobruba-martin** [comment]
Squashed. Thank you

**Pull request closed without merging**